src/styles/ForecastSummaries.css

```diff
@@ -1,4 +0,0 @@
-.forecast-summaries {
-  display: flex;
```

Comment:
if you add `gap: 5px;` it will space out your forecast cards a bit to make it look a bit better